dotenv.sh

```diff
@@ -30,30 +41,38 @@ export_envs() {
 		if is_comment "$key"; then
 			continue
 		fi
+
 		if is_blank "$key"; then
 			continue
 		fi
-		
-		eval val=\""\$$key"\"
-	    if [ -z $val ]; then
+
+		if is_set "$key"; then
+			log_verbose "Existing: $key=$val"
+		else
 			value=$(eval echo "$temp")
 			eval export "$key='$value'";
-	    else
-			if [ "$VERBOSE" = 1 ]; then
-				echo "Existing: $key=$val" >&2
-			fi       
-	    fi
-	done < .env
+		fi
+	done < $1
 }
 
 # inject .env configs into the shell
-if [ -f .env ]; then
-	export_envs
+if [ -f ".env" ]; then
```

Comment:
Do you mean "$DOTENV_FILE"? Since the following `else` output a string contains `$DOTENV_FILE` ?

Reply:
yes :-(